sdk/docs/core/CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 
 ## 1.0.0-preview.4 (Unreleased)
 
+- API breaking changes:
+  - `az_span.h`:
+    - `az_span_init(...)` is renamed to `az_span_create(...)`
```

Comment:
What I've seen somewhat idiomatically - in books and docs, is when talking about functions, you write `f()` and not `f(...)`.

Reply:
I slight prefer the `...`, but will change it.